adrs/0014-avro-support.md

```diff
@@ -50,10 +52,32 @@ We've also added 3 AVRO-specific fields to the `AnyShape` Model via the `AvroFie
 - Aliases
 - Size
 
+### Where we support and DON'T support AVRO Schemas
+We Support AVRO Schemas (inline or inside a `$ref`):
+- as a standalone document or file 
+  - we encourage users to use the `.avsc` file type to indicate that's an avro file, for better suggestions and so on in the platform).
+- inside a message payload in an AsyncAPI
+  - the key `schemaFormat` MUST be declared and specify it's an AVRO payload
 
-For now only parsing is done 
+We don't support AVRO Schemas:
+- inside components --> schemas in an AsyncAPI
+  - because we can't determine if it's an AVRO Schema or any other schema
 
+### AVRO Validation
+We'll use the Apache official plugins for JVM and JS.
 
-## Consequences
 
-None so far.
\ No newline at end of file
+## Consequences / Constraints
+
+The validation plugins differ in interfaces and implementations, and each has some constraints:
+
+### JVM avro validation plugin
+- validation per se is not supported, we try to parse an avro schema and throw parsing results if there are any
+  - this means it's difficult to have location of where the error is thrown, we may give an approximate location from our end post-validation
+- when a validation is thrown, the rest of the file is not being searched for more validations
+  - this is particularly important in large avro schemas, where many errors can be found but only one is shown
+
+
+### Both JVM & JS validation plugins
+- `"default"` values are not being validated when the type is `bytes`, `map`, or `array`
+- the validator treats as invalid an empty array as the default value for arrays (`"default": []`) even though the [Avro Schema Specification](https://avro.apache.org/docs/1.12.0/specification) has some examples with it
```

Comment:
I would add the limitation in validating default and payload in the Shape associated to a field in a record that has a recursive reference

Reply:
Adding that now we are failing, in the future we will try to ignore this cases and show a warning